noq_django/rest_api/api/caseworker_api.py

```diff
@@ -120,3 +121,43 @@ def set_booking_pending(request, booking_id: int):
         return booking
     except BookingStatus.DoesNotExist:
         raise HttpError(404, detail="Booking status does not exist.")
+    
+
+@router.get("/available_all", response=List[ProductSchemaWithPlacesLeft], tags=["host-available"])
```

Comment:
tags should be changed to caseworker-available

Reply:
Done